wasm-transform/Cargo.toml

```diff
@@ -14,5 +14,13 @@ leb128 = "0.2.4"
 anyhow = "1.0.33"
 num_enum = "0.5"
 
+
+[dependencies.concordium-contracts-common]
+# version = "0.4"
+git = "https://github.com/Concordium/concordium-contracts-common.git"
+branch = "validate-contract-and-receive-names"
```

Comment:
And this.

I would prefer if this was not constantly changed. Use the setup that was already provided, by cloning the contracts repo in the provided path. Then nothing needs to be changed in toml files during development.